plugin-dev/Source/Sentry/Private/Tests/SentryEvent.spec.cpp

```diff
@@ -61,6 +61,113 @@ void SentryEventSpec::Define()
 			TestEqual("Fingerprint is empty", OutFingerprint.Num(), 0);
 		});
 	});
+
+	Describe("Event tags", [this]()
+	{
+		It("can be added and removed", [this]()
+		{
+			TestEqual("Tags are empty by default", SentryEvent->GetTags().Num(), 0);
+
+			TestEqual("Can't get any tag while these are empty", SentryEvent->GetTagValue(TEXT("TagsKey0")), TEXT(""));
+
+			SentryEvent->SetTagValue(TEXT("TagsKey1"), TEXT("TagsVal1"));
+			SentryEvent->SetTagValue(TEXT("TagsKey2"), TEXT("TagsVal2"));
+
+			TestEqual("There are two tags adding them as individual items", SentryEvent->GetTags().Num(), 2);
+
+			TestEqual("First tag persist its original value", SentryEvent->GetTagValue(TEXT("TagsKey1")), TEXT("TagsVal1"));
```

Comment:
```suggestion
			TestEqual("First tag retains its original value", SentryEvent->GetTagValue(TEXT("TagsKey1")), TEXT("TagsVal1"));
```
nitpick: description could be more clear (and in similar cases below)